Server/routes/gw2.js

```diff
@@ -46,24 +38,15 @@ router.get('/members', async (req, res) => {
   }
 });
 
-router.put('/members/:memberId', async (req, res) => {
-  const authInfo = await getUserAuthInfo(req);
-  if (!authInfo.authorized) return res.status(403).json('Forbidden');
-
-  const newData = req.body;
-
-  const record = await GuildMember.findOne({ memberId: req.params.memberId });
-  let response = null;
-  if (record) {
-    record.memberId = newData.memberId;
-    record.eventsAttended = newData.eventsAttended;
-    response = await record.save();
-  } else {
-    const newRecord = new GuildMember(...newData);
-    response = await newRecord.save();
-  }
+router.put('/members/:memberId', authenticate, async (req, res) => {
+  const { memberId, eventsAttended } = req.body.newData;
```

Comment:
Forgot to put this in previously.
```suggestion
  const { memberId, eventsAttended } = req.body;
```